frontend/src/utils/apiAxios.ts

```diff
@@ -29,6 +29,8 @@ export const apiAxios = () => {
     if (req.headers && keycloak.authenticated && keycloak.token) {
       req.headers.Authorization = `Bearer ${keycloak.token}`;
     }
+    console.log("req", req);
```

Comment:
this is debug code left in i assume; it shouldn't go in to the repo unless we are gating it with NODE_ENV=development, i think